pkg/ccl/backupccl/testdata/backup-restore/multiregion_mismatch_table_locality

```diff
@@ -0,0 +1,101 @@
+# disabled to run within tenant because multiregion primitives are not supported within tenant
```

Comment:
This isn't strictly true. But I'm fine with us disabling this in mt for simplicity.  This test should work for shared process tenants.